libpod/container.go

```diff
@@ -224,7 +225,7 @@ func (c *Container) setupImageRootfs() error {
 	}
 
 	// TODO SELinux mount label
```

Comment:
Mount label is done, can remove the comment

Reply:
Fixed